tests/playbooks/roles/install-cpo-occm/tasks/main.yaml

```diff
@@ -82,7 +82,7 @@
       fi
       cd $GOPATH/src/k8s.io/cloud-provider-openstack
       # replace image with built image
-      sed -i "s#registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.28.0#{{ remote_registry_host }}/openstack-cloud-controller-manager:${VERSION}#" manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml
+      sed -i "s#registry.k8s.io/provider-os/openstack-cloud-controller-manager:.*#{{ remote_registry_host }}/openstack-cloud-controller-manager:${VERSION}#" manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml
```

Comment:
Hm, this would most likely work, but how about a proper regex instead?

`v[0-9]+\.[0-9]+\.[0-9]+` should be fine?

Reply:
I don't want to do this by purpose, since there could be potential `-alpha`, etc. The wildcard covers everything and you don't need to think about rare cases.

Reply:
Hm, then let's at least use `.+` or even `.+$` here?

Reply:
I'm ok with `.+$`, however in this particular situation it doesn't make sense :see_no_evil:

Reply:
Wait, but `.*`, this will stop working once someone puts image pullspec into quotation marks here: https://github.com/kubernetes/cloud-provider-openstack/blob/f806574b31aa32a269370b3c1a2ed73fc161a498/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml#L41

I understand the intent, but this should be a bit smarter to not bite us in the future.

`[^\'\"]+`?

Reply:
good point. and nice critical thinking ;)